test/unit/test.natural_language_understanding.v1.js

```diff
@@ -38,10 +38,10 @@ describe('natural_language_understanding', function() {
     done();
   });
 
-  it('analyze()', function(done) {
+  it('2017_02_27 version should work', function(done) {
```

Comment:
this should stay `analyze()`, although one more test that uses the old version_date would be nice.